packages/pwa-kit-react-sdk/package.json

```diff
@@ -54,11 +54,12 @@
     "minimatch": "3.0.4",
     "mkdirp": "^1.0.4",
     "prop-types": "^15.6.0",
-    "react-ssr-prepass": "^1.5.0",
     "pwa-kit-runtime": "^2.3.0-dev",
+    "react-ssr-prepass": "^1.5.0",
     "react-uid": "^2.2.0",
     "serialize-javascript": "^6.0.0",
     "svg-sprite-loader": "^6.0.11",
+    "uuid": "^8.3.2",
```

Comment:
Version 9 is available if you want to stay current.

Reply:
I've updated the package to use the latest. 9.0.0